README.md

```diff
@@ -100,3 +103,23 @@ We use [CircleCI](https://circleci.com/gh/nos/client) to automatically create bu
 ## macOS
 1. Create the distributable, i.e. `yarn dist`.
 2. Locate executable file `dist/nOS-1.0.0.dmg`.
+
+
+# Typical installation problems
+
+_Description:_
+
+```
+error An unexpected error occurred: "/Users/nikitin/Development/nos-client/node_modules/node-hid: Command failed.
```

Comment:
In a paragraph prior to this terminal dump, can you explain what led to this output?  Maybe something like:

> Installing dependencies via `yarn install` fails to install prebuilt binaries, resulting in the following error message.